Mobile Buy SDK/Mobile Buy SDK/Models/BUYManagedObject.h

```diff
@@ -25,8 +25,8 @@
 //
 
 #import <CoreData/CoreData.h>
-
-#import <Buy/BUYObject.h>
+#import "BUYObject.h"
```

Comment:
Why did you change the import style here? This form may require additional header search paths, instead of relying on the framework implicit search path.